docs/en/latest/config.json

```diff
@@ -2,8 +2,54 @@
   "version": 2.4,
   "sidebar": [
     {
-      "type": "doc",
-      "id": "architecture-design"
+      "type": "category",
+      "label": "architecture design",
+      "items": [
+        {
+          "type": "doc",
+          "id": "apisix"
```

Comment:
Hi, because we don't have `id` in markdown's meta fields, so we have to use the relative path here.

Please refer to 

```json
{
          "type": "category",
          "label": "General",
          "items": [
            "plugins/batch-requests",
            "plugins/serverless",
            "plugins/redirect",
            "plugins/echo",
            "plugins/server-info"
          ]
}
```

Reply:
Fixed in the new commit.